Lib/sysconfig.py

```diff
@@ -94,6 +95,16 @@
 _CONFIG_VARS = None
 _USER_BASE = None
 
+# Regexes needed for parsing Makefile (and similar syntaxes,
+# like old-style Setup files).
+try:
+    import re
+    _variable_rx = re.compile(r"([a-zA-Z][a-zA-Z0-9_]+)\s*=\s*(.*)")
+    _findvar1_rx = re.compile(r"\$\(([A-Za-z][A-Za-z0-9_]*)\)")
+    _findvar2_rx = re.compile(r"\${([A-Za-z][A-Za-z0-9_]*)}")
+except ImportError:
+    pass
```

Comment:
Alternative idea: have the regexes as global string literals, then use simple `re.search` function where needed and benefit from its existing cache.  I don’t think this is critical code called many times (but could be wrong!).

Reply:
Good idea. I'm gonna try to implement it. It should not cause any harm because those variables are private :thinking:

Reply:
Implemented.